swan-lake/by-example/alternate-wait/content.jsx

```diff
@@ -1,17 +1,13 @@
-import React, { useState, useEffect, createRef } from "react";
-import { setCDN } from "shiki";
+import React, { useState, createRef } from "react";
```

Comment:
Changes on `content.jsx` files get replaced next time we do a BBE gen. `Content.jsx` file is created using https://github.com/ballerina-platform/ballerina-dev-website/blob/master/.github/scripts/bbe/convertMarkdown.js. You can do the changes there. Also check https://github.com/ballerina-platform/ballerina-dev-website/blob/master/utils/bbe.js. It is used to get the theme and the language for BBEs.

Reply:
@sm1990 please review https://github.com/ballerina-platform/ballerina-dev-website/blob/35d2e2e3df476dbc99de6bbed145d37c71f40408/.github/scripts/bbe/convertMarkdown.js